cpuid/src/bit_helper.rs

```diff
@@ -0,0 +1,188 @@
+/// Structure representing a range of bits in a number
+///
+pub struct BitRange {
+    /// most significant bit
+    pub left: u32,
```

Comment:
I'm missing what `left` and `right` are used for? How is 'a range of bits' represented here exactly?

Reply:
Done